src/v0/destinations/fb_custom_audience/recordTransform.js

```diff
@@ -269,10 +269,11 @@ const processRecordInputsV2 = (groupedRecordInputs) => {
 
 function processRecordInputs(groupedRecordInputs) {
   const event = groupedRecordInputs[0];
```

Comment:
we got product approval to discontinue support of audience destinations for Eventstream - we can handle that in the next release post docs update & any comms rollout